.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+For bug reports, please respect the following guidelines and check the boxes accordingly.
+
+For questions of the type: "How can this be done?", please consider posting them on Stackoverflow.
+
+For everything else ignore this template.
+
+## Bug report
+
+- [ ] Start a new R session
```

Comment:
possibly ask people to use `--vanilla` or something? I don't know how it looks in RStudio, but if people use the command line R, they might have some `.RData` which they overlook.